reconcile/test/test_gitlab_labeler.py

```diff
@@ -0,0 +1,142 @@
+from unittest import TestCase
```

Comment:
we have agreed into writing tests with pytest. It looks like a lot of work but actually, it isn't... with very little effort you can transform this into a pure pytest test. See https://github.com/app-sre/qontract-reconcile/pull/2077 as example